app/views/local_transaction/results.html.erb

```diff
@@ -19,13 +19,11 @@
           href: @interaction_details['local_interaction']['url'],
         } %>
       </p>
-    </div>
-  <% elsif @location_error && @location_error.no_location_interaction? %>
-    <div class="interaction">
-      <p class="govuk-body">
-        <%= t(@location_error.message, **@location_error.message_args) %>
-      </p>
+    <% else %>
       <% if @local_authority.url.present? %>
+        <p class="govuk-body">
+          We do not know if they offer this service. Search their website to find out if they do, or what other services they offer.
+        </p>
```

Comment:
Should this be in en.yml?

Reply:
The precedence for this file seems to be hardcoding text (I actually removed the only usage of `t` from it)

Reply:
👍